charter/index.md

```diff
@@ -0,0 +1,211 @@
+
+## Scope
+
+The Font and Text community group gathers individuals and organizations
+interested in discussing and developing specifications and implementations
+for technologies such as shaping and layout
+which operate on and at the interface between
+text encoding and font formats.
+
+Under the present charter, the Community Group will
+determine a structure and process
+under which such specifications may be created
+by which the greatest number of interested parties may be able to participate.
+Until such a structure and process is determined,
+no technical development of specifications or implementations
+will be undertaken by this Community Group
+and no technical contributions will be accepted.
+
+## Deliverables
+
+* This Community Group aims to produce a proposed Charter under which
+this community should conduct its technical work,
+either to be used as an updated Charter for this CG,
+or to set up a different group,
+possibly a W3C Working Group,
+to conduct that work.
+
+* If the Community Groups determines that it would be appropriate
+to divide the technical work into several venues,
+it may produce several such proposed charters
+rather than a single one.
+
+* Various supporting documents and articles
+describing the problem space
+may also be produced.
+
+All deliverables of this Community Group will use the
+[W3C Software and Document License](http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document).
+
+This group maintains an [index](https://font-text.github.io/work-items/)
+of the documents it is working on.
+
+## Communication
+
+This group operates under the
+[W3C Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/).
+
+This group will conduct its work publicly,
+primarily using the public-font-text@w3.org mailing list
+([archives](https://lists.w3.org/Archives/Public/public-font-text/)
+for minutes, announcements, and high level discussions,
+and [GitHub](https://github.com/font-text)
+for detailed discussions
+(See [Contribution Mechanics](#contribution-mechanics).
+
+Teleconferences and face-to-face meetings
+may be held from time to time,
+provided sufficient advance notice is given.
+Minutes of any such meeting will be
+posted to the group’s public mailing list
+in a timely manner,
+and any resolution will be called out explicitly
+(See [Decision Policy](#decision-policy)).
+
+Deliverables and supporting documents
+produced by this Community Group are
+hosted on the group’s [GitHub](https://github.com/font-text).
+
+The group uses the private internal-font-text@w3.org mailing list ([archives](https://lists.w3.org/Archives/Member/internal-font-text/))
+for administrative purposes,
+such as organizing meetings,
+requesting commit access to GitHub,
+etc.
+
+Additionally, the group will use
+the #font-text channel on the [W3C IRC server](https://www.w3.org/Project/IRC/)
+during meetings
+and for day-to-day casual chatting.
+
+## Decision Policy
```

Comment:
In comparing this to https://www.w3.org/Fonts/WG/webfonts-2018.html#decisions I'm curious if we can/should structure this similarly or if there are specific reasons we do not wish to do that?

Reply:
I gather what was in Simon's draft came directly from [this w3c template](https://github.com/w3c/cg-charter). 

@rsheeter: I see they're different; what do you think are the most significant differences?

Reply:
Tbh the biggest concern for me was I saw no citation and that made me worry we'd just made something up rather than using something well established.

Reply:
The webfonts statement on decisions seems clearer wrt process.

Reply:
This draft came from a template provided by Florian at the W3C. Each CG can choose how it operates, so rather than which template to choose, the real question is how we wish to operate. What are your thoughts on the specific differences?

Reply:
The relevant §6 from the webfonts charter example is a bit more specific about things like comment periods venues. I'm not sure we're ready for that level of specificity yet. As a provisional draft charter to get us bootstrapped I think being a little more vague might be fine. We can establish what works for this group as it becomes more clear what the scope is.

The one thing I do really like f from the webfonts charter is the 3rd paragraph that makes all decisions made in face-to-face (or virtual) synchronous meetings provisional with a small repose period for people that couldn't participate live to have a say. The objection period wouldn't have to be long (10 days maybe, not 3 months) and it wouldn't have to be as long as for other fully async proposals that might move slower, but it would take some of the pressure off and avoid "show up to this meeting or loose your vote on this issue".

Reply:
W3C has different groups operating on different modes.
* Most groups allow decisions to be made asynchronously, some at the chairs discression, some with an explicit "call for consensus" process and an explicit time out.
* Some groups consider any decision made synchronously to be provisional (with an explicit period to disagree), other consider synchronous meetings to be the main way actual decisions get made.

Personally, I prefer synchronous decisions to be immediately valid, rather than tentative:
* Either way, any decision can (and should) be reopened when new information/arguments are brought forward, regardless of how much time has elapsed, so we're never stuck with a bad decision just because someone wasn't there the right day (external constraints such as compatibility concerns or unchangeable implementations may force our hand, but that's independent of the decision policy). And it is a good chairing practice to defer discussion of a topic to a later meeting if key participants are missing, to avoid making fragile decisions in the first place.
* As an Editor, I find that keeping track of which decisions have been tentatively made but not yet confirmed before making edits ads unwelcome mental overhead. I have no problem making an edit and reverting it later if we change our mind, remembering to do things with a 10 days delay is annoying. It doesn't necessarily have to be the editor (could be the chairs) who keep track of where we are, but still, that's overhead I'd rather not have.
* The previous point is made worse when you have a set of interdependent decisions to make. Some times what you do for problem B depends on the decision for problem A. If everything is up in the air, it's hard to lock it all down.

All in all, if any decision was final, I think it would be important to have delays to make sure we don't miss out on anyone, but given than everything can be reopened, I think it ads unnecessary bureaucracy.

That said, it's all a matter of preference, and there are successful groups operating under either model. I won't object either way.

Reply:
Thanks Florian, that does shed some light on options and I'm at least partially convinced.

Reply:
>I'm not sure we're ready for that level of specificity yet. 

That's fair: this is still very nacent.

@frivoal: well said

Reply:
Please remember that the scope of the current charter is basically "discuss what the next charter is going to look like". If you want a stricter decision policy for *technical* decisions, that will need to go into the following charter, but may not need to go into this one.

@rsheeter: Does Florian's explanation address your concerns or do you want to propose a replacement here?

Reply:
Yes, thank you all for elaborating, I think I'm good. I suggest citing the template since it sounds like we used one

Reply:
My understanding is that this discussion has run its course. I'm going to mark it as resolved. It's been discussed where the initial template came from, but we've also heavily modified it to suit our needs and plans as Simon noted at the top of this thread. I don't think citing the boilerplate we started from in the final version would be of much utility because it wouldn't be the same nor have any effect on it, only the charter as posted here really has any bearing in the end.

If there are any unresolved concerns I've missed that you think require action or edits, please do start a new thread (with an edit suggestion if possible). We're in the final 10 days before this gets ratified if there are no further objections.